main.py

```diff
@@ -75,39 +75,43 @@ def rest_get(self, namespace, start=None, end=None, method=None,
     @listen_to('kytos.kronos.save')
     def event_save(self, event):
         """Save the data in one of the backends."""
+        error = None
+        result = None
+
         try:
             self.backend.save(event.content['namespace'],
                               event.content['value'],
                               event.content['timestamp'])
-            result = 'Value saved.'
-            error = None
         except (NamespaceError, ValueConvertError) as exc:
-            result = None
             error = (str(exc), exc.__class__.__name__)
```

Comment:
With this change, `result` is always `None`. It should be set when successfull.

Reply:
Which value it should be set?

Reply:
Just a message reporting success.

Reply:
I added the message to save method and delete. Thanks.